lib/lookup.json

```diff
@@ -333,117 +363,122 @@
   },
   "multer": {
     "prefix": "v",
-    "skip": "win32",
+    "skip": ["darwin", "win32"],
     "maintainers": "linusu"
   },
   "nan": {
     "maintainers": ["nodejs/addon-api"],
     "prefix": "v",
     "scripts": ["rebuild-tests", "test"],
     "tags": "native",
+    "skip": ["darwin"],
     "head": true
   },
   "node-gyp": {
     "envVar": { "FAST_TEST": "true" },
     "prefix": "v",
     "tags": "native",
     "maintainers": ["nodejs/node-gyp"],
-    "skip": ["win32"]
+    "skip": ["darwin", "win32"]
   },
   "node-sass": {
     "prefix": "v",
     "tags": "native",
     "maintainers": "xzyfer",
     "comment": "Flaky because of test timeouts",
-    "skip": true
+    "skip": [true, "darwin"],
   },
   "npm": {
     "maintainers": ["nodejs/npm"],
     "prefix": "v",
-    "skip": ["aix", "s390"]
+    "skip": ["darwin", "aix", "s390"]
   },
   "path-to-regexp": {
     "prefix": "v",
     "maintainers": "blakeembrey",
-    "skip": ["aix"]
+    "skip": ["darwin", "aix"]
   },
   "pino": {
     "prefix": "v",
     "maintainers": "mcollina",
-    "skip": ["aix", "s390x"]
+    "skip": ["darwin", "aix", "s390x"]
   },
   "prom-client": {
     "prefix": "v",
+    "skip": ["darwin"],
     "maintainers": ["siimon", "zbjornson", "SimenB"]
   },
   "pug": {
     "prefix": "pug@",
     "yarn": true,
     "maintainers": ["tj", "ForbesLindesay"],
-    "skip": ["win32", "aix", ">=16"],
+    "skip": ["darwin", "win32", "aix", ">=16"],
     "comment": "Error message changes in V8 9.3",
     "repo": "https://github.com/pugjs/pug"
   },
   "pump": {
     "prefix": "v",
     "maintainers": "mafintosh",
-    "skip": "win32"
+    "skip": ["darwin", "win32"]
   },
   "pumpify": {
     "prefix": "v",
     "maintainers": "mafintosh",
-    "skip": "win32"
+    "skip": ["darwin", "win32"]
   },
   "q": {
     "prefix": "v",
     "maintainers": "kriskowal",
-    "skip": "win32"
+    "skip": ["darwin", "win32"]
   },
   "react": {
     "prefix": "v",
     "maintainers": ["acdlite", "gaearon", "sophiebits"],
     "yarn": true,
-    "skip": [">=12", "aix", "s390", "ppc"]
+    "skip": ["darwin", ">=12", "aix", "s390", "ppc"]
   },
   "readable-stream": {
     "prefix": "v",
     "flaky": ["ppc"],
     "maintainers": "nodejs/streams",
-    "skip": [">=12", "win32"]
+    "skip": ["darwin", ">=12", "win32"]
   },
   "ref": {
     "flaky": "aix",
     "tags": "native",
     "maintainers": "TooTallNate",
-    "skip": [true, "12", "win32"]
+    "skip": [true, "12", "win32", "darwin"]
   },
   "resolve": {
     "prefix": "v",
     "maintainers": "ljharb",
+    "skip": ["darwin"],
```

Comment:
this is fine if needed, but resolution in particular varies across platforms, so it'd be nice to keep this one if it's not flaky/overly slow.

Reply:
done in 6700881